Kubernetes/PublicBrowse/dev-publicbrowse-kubernetes-manifest.yml

```diff
@@ -30,68 +28,92 @@ spec:
           limits:
             cpu: 500m
         env:
+
+        - name: BASE_URL
+          value: ""
+
         - name: LOGGER_LEVEL
-          value: "info"
+          value: srt-loggerlevel@azurekeyvault
+
+          ## Boolean to indicate if we show the login/logout component
         - name: LOGIN_ENABLED
           value: "true"
+
+          ## Boolean to indicate if we show the capabilities selector page
         - name: USE_CAPABILITIES_SELECTOR
           value: "true"
+
+          ## Boolean to indicate if we show the COVID-19 feature
         - name: SHOW_COVID19
           value: "true"
+
+          ## Node environment
         - name: NODE_ENV
-          value: "production"
+          value: "development"
```

Comment:
might need @alibombaye to advise on this. Development to the UI guys in this context might mean local and do stuff with the app.

Reply:
Spoke with @alibombaye about this before. For the Dev environment we want to set the value as "development", but for Prod it will need to be "production"